app/javascript/hooks/useScript.tsx

```diff
@@ -13,7 +13,7 @@ const useScript = (src) => {
       }
       // Fetch existing script element by src
       // It may have been added by another intance of this hook
-      let script = document.querySelector(`script[src="${src}"]`)
+      let script: any = document.querySelector(`script[src="${src}"]`)
```

Comment:
This is probably a quick fix to appease the linter, but we should try to avoid `any` types in general.

I realize that this file isn't great about typing things, and working to overhaul it is outside the scope of this ticket.

Reply:
Ugh, I completely agree with you. I'm not a fan of any types. This was a quick fix to make the linter happy since we were effectively treating `script` (and below `selectInDom`) as any type anyway. 

I will work on this today to see if I can get the typing right, if I can't then I will create a ticket to tackle it later.

Reply:
Created a ticket to track this effort: https://sfgovdt.jira.com/browse/DAH-1581